src/snapshots/ruff__linter__tests__B008_B006_B008.py.snap

```diff
@@ -0,0 +1,125 @@
+---
```

Comment:
Comparison:

```
> cargo run -- --no-cache --select B008 resources/test/fixtures/B006_B008.py
resources/test/fixtures/B006_B008.py:85:61: B008 Do not perform function calls in argument defaults.
resources/test/fixtures/B006_B008.py:89:64: B008 Do not perform function calls in argument defaults.
resources/test/fixtures/B006_B008.py:93:60: B008 Do not perform function calls in argument defaults.
resources/test/fixtures/B006_B008.py:109:39: B008 Do not perform function calls in argument defaults.
resources/test/fixtures/B006_B008.py:113:12: B008 Do not perform function calls in argument defaults.
resources/test/fixtures/B006_B008.py:113:32: B008 Do not perform function calls in argument defaults.
resources/test/fixtures/B006_B008.py:117:30: B008 Do not perform function calls in argument defaults.
resources/test/fixtures/B006_B008.py:155:34: B008 Do not perform function calls in argument defaults.
resources/test/fixtures/B006_B008.py:160:30: B008 Do not perform function calls in argument defaults.
resources/test/fixtures/B006_B008.py:164:45: B008 Do not perform function calls in argument defaults.
resources/test/fixtures/B006_B008.py:170:21: B008 Do not perform function calls in argument defaults.
resources/test/fixtures/B006_B008.py:170:31: B008 Do not perform function calls in argument defaults.
resources/test/fixtures/B006_B008.py:176:22: B008 Do not perform function calls in argument defaults.
resources/test/fixtures/B006_B008.py:181:19: B008 Do not perform function calls in argument defaults.
resources/test/fixtures/B006_B008.py:181:37: B008 Do not perform function calls in argument defaults.
```

```
> flake8 --select B008 resources/test/fixtures/B006_B008.py
resources/test/fixtures/B006_B008.py:85:61: B008 Do not perform function calls in argument defaults.
resources/test/fixtures/B006_B008.py:89:64: B008 Do not perform function calls in argument defaults.
resources/test/fixtures/B006_B008.py:93:60: B008 Do not perform function calls in argument defaults.
resources/test/fixtures/B006_B008.py:109:39: B008 Do not perform function calls in argument defaults.
resources/test/fixtures/B006_B008.py:113:12: B008 Do not perform function calls in argument defaults.
resources/test/fixtures/B006_B008.py:113:32: B008 Do not perform function calls in argument defaults.
resources/test/fixtures/B006_B008.py:117:30: B008 Do not perform function calls in argument defaults.
resources/test/fixtures/B006_B008.py:160:30: B008 Do not perform function calls in argument defaults.
resources/test/fixtures/B006_B008.py:164:45: B008 Do not perform function calls in argument defaults.
resources/test/fixtures/B006_B008.py:170:21: B008 Do not perform function calls in argument defaults.
resources/test/fixtures/B006_B008.py:170:31: B008 Do not perform function calls in argument defaults.
resources/test/fixtures/B006_B008.py:176:22: B008 Do not perform function calls in argument defaults.
resources/test/fixtures/B006_B008.py:181:19: B008 Do not perform function calls in argument defaults.
resources/test/fixtures/B006_B008.py:181:37: B008 Do not perform function calls in argument defaults.
```

Reply:
`resources/test/fixtures/B006_B008.py:155` is:

https://github.com/charliermarsh/ruff/blob/fbdc075e5ce76037c80358f842478f865447c3ae/resources/test/fixtures/B006_B008.py#L155